tests/Sabre/Xml/ContextStackTest.php

```diff
@@ -15,11 +15,11 @@
  */
 class ContextStackTest extends TestCase
 {
-    private $stack;
+    private ContextStack $stack;
 
     public function setUp(): void
     {
-        $this->stack = $this->getMockForTrait('Sabre\\Xml\\ContextStackTrait');
+        $this->stack = new ContextStack();
```

Comment:
Ref: https://github.com/phpstan/phpstan-phpunit/issues/30

It seems better to use a real class that is just a "wrapper" around the trait, and test that class. This "just works"  for the tests here, they still pass. And it makes it possible to declare the type of `$stack`